src/js/components/commons/icons/index.js

```diff
@@ -1,5 +1,3 @@
-export { default as GrafanaIcon } from './grafana.icon';

```

Comment:
If we stop exporting them then we probably don't use them anymore. We should take this opportunity to delete the corresponding .icon files.

Reply:
I can do that to Renaud !